src/commands/generate.cjs

```diff
@@ -32,9 +32,9 @@ exports.handler = args => {
     args.logger = () => {};
   }
 
-  generate(args);
+  
 };
-
+generate(path.joinSafe(__dirname, '../../p5/docs/reference/data.json'), path.joinSafe(__dirname, '../../DefinitelyTyped/types/p5'));
```

Comment:
This hard-codes the data path. That can be OK but there's a fair bit of clean-up to do if you want to go that way. Fastest for you would probably be to make the output path an argument in the download command.

Reply:
I guess there are 3 ways:
1. use variables, and put them all in a js file. <= current
2. put the configuration in a JSON file. <= similar to 1, but less type-safety.
3. use CLI command arguments.